src/main/resources/view/CandidateListCard.fxml

```diff
@@ -27,6 +27,7 @@
         </Label>
         <Label fx:id="name" text="\$first" styleClass="cell_big_label" />
       </HBox>
+      <FlowPane fx:id="statusPane" />
```

Comment:
Just a minor suggestion, would it be relevant to include the `prefWrapLength` for the tags! :) - or perhaps you can look into this when the three bottom panels are up as well!

Reply:
Thanks for noting it out !! will make the changes now !